itc4j-jackson/src/test/java/itc4j/SerializationTest.java

```diff
@@ -0,0 +1,38 @@
+package itc4j;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Ian Eure <ian.eure@gmail.com>
+ */
+public class SerializationTest {
```

Comment:
@ieure Did you consider writing a test that compares the result of serialization against an expected string value?
This would serve as documentation for the result of the serialization, so one could know what to expect from it.